### ISSUE TITLE
[regression] fatal error: 'EABase/eabase.h' file not found

### ISSUE THREAD

**Issue by yurivict** [electronicarts/EASTL#296]
```3.14.00``` fails:
```
In file included from /usr/ports/devel/eastl/work/EASTL-3.14.00/source/hashtable.cpp:6:
/usr/ports/devel/eastl/work/EASTL-3.14.00/include/EASTL/internal/hashtable.h:30:10: fatal error: 'EABase/eabase.h' file not found
#include <EABase/eabase.h>
         ^~~~~~~~~~~~~~~~~
1 error generated.
```

FreeBSD 12 amd64
Built from the port.

**Comment by rparolin** [comment]
The project changed to use git submodules for its dependent packages.

**Comment by yurivict** [comment]
It fails to find the headers because the include paths are missing, not because modules are missing:
```
[8/10] /usr/bin/c++  -DEASTL_EXPORTS -DEASTL_OPENSOURCE=1 -D_CHAR16T -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS -I/usr/ports/devel/eastl/work/EASTL-3.14.00/include -O2 -pipe -fno-omit-frame-pointer -fstack-protector-strong -fno-strict-aliasing -fno-omit-frame-pointer -std=c++17 -std=c++14 -O2 -pipe -fno-omit-frame-pointer -fstack-protector-strong -fno-strict-aliasing -fno-omit-frame-pointer -fPIC -MD -MT CMakeFiles/EASTL.dir/source/red_black_tree.cpp.o -MF CMakeFiles/EASTL.dir/source/red_black_tree.cpp.o.d -o CMakeFiles/EASTL.dir/source/red_black_tree.cpp.o -c /usr/ports/devel/eastl/work/EASTL-3.14.00/source/red_black_tree.cpp
FAILED: CMakeFiles/EASTL.dir/source/red_black_tree.cpp.o 
/usr/bin/c++  -DEASTL_EXPORTS -DEASTL_OPENSOURCE=1 -D_CHAR16T -D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS -I/usr/ports/devel/eastl/work/EASTL-3.14.00/include -O2 -pipe -fno-omit-frame-pointer -fstack-protector-strong -fno-strict-aliasing -fno-omit-frame-pointer -std=c++17 -std=c++14 -O2 -pipe -fno-omit-frame-pointer -fstack-protector-strong -fno-strict-aliasing -fno-omit-frame-pointer -fPIC -MD -MT CMakeFiles/EASTL.dir/source/red_black_tree.cpp.o -MF CMakeFiles/EASTL.dir/source/red_black_tree.cpp.o.d -o CMakeFiles/EASTL.dir/source/red_black_tree.cpp.o -c /usr/ports/devel/eastl/work/EASTL-3.14.00/source/red_black_tree.cpp
In file included from /usr/ports/devel/eastl/work/EASTL-3.14.00/source/red_black_tree.cpp:27:
/usr/ports/devel/eastl/work/EASTL-3.14.00/include/EASTL/internal/config.h:61:11: fatal error: 'EABase/eabase.h' file not found
        #include <EABase/eabase.h>
                 ^~~~~~~~~~~~~~~~~
1 error generated.
ninja: build stopped: subcommand failed.
===> Compilation failed unexpectedly.
Try to set MAKE_JOBS_UNSAFE=yes and rebuild before reporting the failure to
the maintainer.
*** Error code 1

Stop.
make: stopped in /usr/ports/devel/eastl
[yuri@yv /usr/ports/devel/eastl]$ find . -name eabase.h
./work/EASTL-3.14.00/test/packages/EABase/include/Common/EABase/eabase.h
```

**Comment by Zoodinger** [comment]
@yurivict I have the same issue. As a workaround, try enabling EASTL_BUILD_TESTS.

I opened a new issue here seeing how this one got closed and abandoned without a resolution, but I'm still waiting for a proper fix:
https://github.com/electronicarts/EASTL/issues/320
